### DISCUSSION TITLE
Hello how do i drop/delete a table in migration

### DISCUSSION THREAD

**Discussion by Samamoah** [Nozbe/WatermelonDB#1293] [Q&A]
How do i delete/drop a table in migration.

its seems Schema/migration exports only these but not deleting/dropping a table

` export function schemaMigrations(migrationSpec: SchemaMigrationsSpec): SchemaMigrations
  export function createTable(tableSchemaSpec: TableSchemaSpec): CreateTableMigrationStep
  export function addColumns({
    table,
    columns,
  }: { table: TableName<any>, columns: ColumnSchema[] }): AddColumnsMigrationStep`

**Comment by KrisLau** [comment]
https://nozbe.github.io/WatermelonDB/Advanced/Migrations.html#migration-steps

> Other types of migrations (e.g. deleting or renaming tables and columns) are not yet implemented. See [migrations/index.js](https://github.com/Nozbe/WatermelonDB/blob/master/src/Schema/migrations/index.js). Please contribute!

**Reply by Jenspi** [reply]
Still the case in 2025, I'm guessing? 🥹

**Reply by alexrififi** [reply]
> Still the case in 2025, I'm guessing? 🥹

yep

**Reply by rjmatthews62** [reply]
unsafeExecuteSql exists.,.. presumably you could do a "drop table" in there....?
